src/components/InfoMap.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement, useState } from "react";
+import {
+  GoogleMap,
+  InfoWindow,
+  LoadScript,
+  Marker,
+} from "@react-google-maps/api";
+import styled from "styled-components";
+
+function InfoMap(props: any): ReactElement {
+  const mapStyles = {
+    height: "100%",
+    width: "100%",
+  };
+
+  const defaultCenter = {
+    lat: 41.3851,
+    lng: 2.1734,
+  };
+  const [selected, setSelected] = useState<any>({});
+
+  const onSelect = (item: any) => {
+    setSelected(item);
+  };
+
+  return (
+    <InfoMapContainer>
+      <LoadScript googleMapsApiKey=''>
+        <GoogleMap
+          mapContainerStyle={mapStyles}
+          zoom={13}
+          center={defaultCenter}
+          options={{ gestureHandling: "greedy" }}
```

Comment:
is it required ?

Reply:
I added this to remove zooming by command  + scroll